Makefile

```diff
@@ -1,7 +1,12 @@
+.PHONY: init test
 init:
-    pip install -r requirements.txt
-
+	pip install pipenv --upgrade
+	pipenv install --dev

```

Comment:
Just mention this in `Readme.md` would be sufficient.

Reply:
I can write this on `RAEDME`  like 
```
## setup project
$ make init
```

Reply:
I am mainly inspired by this doc(https://docs.python-guide.org/writing/structure/#makefile)

Reply:
Ditto. This will select pip + python2. pipenv + python2 will be installed if your system uses python2 as a default.

Reply:
I changed it.